nbodykit/measurestats.py

```diff
@@ -481,7 +481,7 @@ def project_to_basis(comm, x3d, y3d, edges, los='z', poles=[], symmetric=True):
                     weighted_y3d.real[:, nonsingular] *= 2.
                     weighted_y3d.imag[:, nonsingular] = 0.
                     
-            weighted_y3d *= 0.5*(2*ell + 1.)
+            weighted_y3d *= (2.*ell + 1.)
```

Comment:
I don't think we need this factor of 1/2 since we are already averaging mu and -mu into one bin -- agree, @rainwoodman ?